general/administration/installing.md

```diff
@@ -240,11 +240,11 @@ Create four sub-directories for Jellyfin data.
 sudo mkdir data cache config log
 ```
 
-If you are running Debian or a derivative, you can also download and install an ffmpeg release built specifically for Jellyfin. Be sure to download the latest release that matches your OS (4.2.1-3 for Debian Stretch assumed below).
+If you are running Debian or a derivative, you can also download and install an ffmpeg release built specifically for Jellyfin. Be sure to download the latest release that matches your OS (4.2.1-4 for Debian Stretch assumed below).
```

Comment:
this is going to be out of date soon, can we make a more stable instruction page that doesn't need updating each time we publish a new package?

Reply:
I'd rather switch this whole line to encourage either (a) jellyfin-ffmpeg for Debuntu, or (b) "The latest 4.1 FFMpeg release from your distribution". No need to go into the -X version numbers or anything.

-4 is already out of date ;-)

Reply:
I added the repo link so it's easier to find the appropriate package.

Reply:
also updated the example to .5 :P